frontend/src/pages/api/auth/[...nextauth].ts

```diff
@@ -0,0 +1,35 @@
+import NextAuth from "next-auth"
```

Comment:
Question: I am not familiar with this framework, but do we need adding special chars in the filename?

Reply:
Yes, because this is a route file and the file name is dynamic, depending on the request the value changes. 
Check the documentation here: [Link](https://next-auth.js.org/getting-started/example)